src/OrchardCore.Modules/OrchardCore.Seo/GraphQL/SeoMetaQueryObjectType.cs

```diff
@@ -0,0 +1,67 @@
+﻿using GraphQL.Types;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Media.Fields;
+using OrchardCore.ResourceManagement;
+using OrchardCore.Seo.Models;
+
+namespace OrchardCore.Seo.GraphQL;
+
+public class SeoMetaQueryObjectType : ObjectGraphType<SeoMetaPart>
```

Comment:
This should be the only thing required, since we just need to expose the properties of this part for a content item to be listed and accessed.